### PR TITLE
In-transit configuration for provider mode

### PR DESCRIPTION
**Pull request by fbalak**

### CODE REVIEW

**Comment by fbalak** [comment]
Verified: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/41142/

File: ocs_ci/deployment/encryption.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    configuration.
+
+    Args:
+        cluster_data (dict): storage cluster YAML data
+
+    Returns:
+        dict: updated storage storage cluster yaml
+    """
+    if config.ENV_DATA.get("encryption_at_rest"):
+        logger.info("Enabling encryption at REST!")
```
Comment (dahorak) [inline ocs_ci/deployment/encryption.py:47]: Small inconsistency in the log level - here is `info` while for *in-transit encryption* is used `debug`.

**Review by dahorak — APPROVED** [review]
Just small note, overall LGTM

**Review by dahorak — APPROVED** [review]

**Review by jilju — APPROVED** [review]

**Review by DanielOsypenko — APPROVED** [review]

**Pull request merged**